app/src/main/java/io/github/cmput301f19t19/legendary_fiesta/MainActivity.java

```diff
@@ -29,6 +29,10 @@ protected void onCreate(Bundle savedInstanceState) {
         NavigationUI.setupActionBarWithNavController(this, navController, appBarConfiguration);
         NavigationUI.setupWithNavController(navView, navController);
 
+        //hide action bar
+        ActionBar actionBar = getSupportActionBar();
+        if(actionBar != null)
```

Comment:
You can actually hide it in the `styles.xml`

Reply:
tried it, but doesn't work. This is what i had in my styles.

<style name="NoActionBar" parent="@style/AppTheme">
    <item name="android:windowActionBar">false</item>
    <item name="android:windowNoTitle">true</item>
</style>

Reply:
>tried it, but doesn't work. This is what i had in my styles.
>
><style name="NoActionBar" parent="@style/AppTheme"> false true </style>

Well, this is not how it works. See https://github.com/CMPUT301F19T19/legendary-fiesta/pull/33/files#diff-6f870d7e261f41dd3e52a2a129fa1fc5R10-R13 for an example.

Reply:
You can try implementing this method in the future, I'm not quite sure how it works. It uses .setTheme() through the Auth.getInstance() method.